tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -144,6 +145,10 @@ def visible_apps(self):
         return [self.InstalledApp(self.marionette, root_element)
                 for root_element in self.app_elements if root_element.is_displayed()]
 
+
+    def wait_for_homescreen_visible(self):
+        self.wait_for_element_displayed(*self._homescreen_container_locator)
```

Comment:
We're moving away from using the wait_for_ methods, please use Wait and expected from Marionette.